docs/wiki/index.md

```diff
@@ -1,5 +1,11 @@
-Welcome to the niri wiki!
+Welcome to the niri documentation!
 
-Check out the available pages on the right.
+Feel free to look through usage and [Getting started](./Getting-Started.md).
+If you're looking for ways to configure niri, check out the [introduction to configuration](./Configuration:-Introduction.md).
+If you want to contribute to niri, look at the development section and [how to develop niri](./Development:-Developing-niri.md).
 
-The wiki is open to contribution, but please discuss bigger changes in [our Matrix room](https://matrix.to/#/#niri:matrix.org) first! The wiki is generated from files in the `docs/wiki/` folder of the repository, so you can open a pull request modifying it there.
+If you're not already here, check out [our new documentation pages](https://yalter.github.io/niri/)!
```

Comment:
```suggestion
If you're not already here, check out our new wiki website! https://yalter.github.io/niri/
```
- It's clearer where is "here" with an explicit link
- I like "wiki website" more than "documentation pages"